### DISCUSSION TITLE
PWA Support

### DISCUSSION THREAD

**Discussion by vedxp** [vuejs/vitepress#1417] [Q&A] [answered]
As documented [here](https://vite-plugin-pwa.netlify.app/), this
```
import { VitePWA } from 'vite-plugin-pwa'
export default defineConfig({
  plugins: [
    VitePWA({ registerType: 'autoUpdate' })
  ]
})
```

needs to be configured in `vite.config.js` / `vite.config.ts` file.

But there's no such file in VitePress scaffolded site.

So, I need help configuring pwa.

**Comment by brc-dd** [comment, marked as answer]
> But there's no such file in VitePress scaffolded site.

Create one.

**Reply by vedxp** [reply]
Yeah sure, I'll do the same.
Thank You!

**Reply by vedxp** [reply]
@brc-dd 

Should `vite.config.js` be placed in root `dir` or any other `dir`?

**Reply by brc-dd** [reply]
If you're running `vitepress build docs` then it should be inside `docs`, if you're simply running `vitepress build` then it should be in your current working directory.

**Reply by vedxp** [reply]
Okay sir, thanks for the help!
